modules/pool/pool.gql

```diff
@@ -1,5 +1,5 @@
 extend type Query {
-    poolGetPool(id: String!): GqlPoolBase!
+    poolGetPool(id: String!, chain: GqlChain): GqlPoolBase!
```

Comment:
Do we need a chain filter when searching by ID? It's already unique, isn't it? same in poolGetSnapshots, poolJoinExit, userGetPoolJoinExits, userGetSwaps

Reply:
at least it _should_ be unique, yes. No strong opinion, but looks like the safer way